patches/minecraft/net/minecraft/entity/player/ServerPlayerEntity.java.patch

```diff
@@ -193,10 +207,11 @@
           this.field_70170_p.func_217376_c(itementity);
           ItemStack itemstack = itementity.func_92059_d();
           if (p_146097_3_) {
-@@ -1403,6 +_,15 @@
+@@ -1402,6 +_,29 @@
+          }
  
           return itementity;
-       }
++      }
```

Comment:
What is this change?

Reply:
The `genPatches` task generated it like this. It thought I added the bracket here and interpreted the bracket at the end of the `refreshTabListName` method as if it has been there before. But I don't think this should cause any problems.

Reply:
There's a whitespace difference here. There should be a space after the bracket.

Reply:
Sorry but I don't seem to be getting the point. When I add a space after that bracket there will just be an additional pointless space in the patch file that was not there before and the change will still be there.

Reply:
Your change is removing the space. That's why this line in the patch exists.
![image](https://user-images.githubusercontent.com/42079760/116813051-c6ba6a00-ab49-11eb-9e5b-7a8acd38fe60.png)

Reply:
Isn't there an extra space in patch files for unmodified lines?

Reply:
If a line is unmodified it won't show up as a modified line..

Reply:
![image](https://user-images.githubusercontent.com/63002502/116813196-ebffa600-ab52-11eb-987b-1182ce470589.png)
When I add a spce in the patch file and run `setup` `ServerPlayerEntity.java` looks like this. As you can see here, the bracket is offset by one to the vertical line. So I can change this but for me it gives wrong formatting.

Reply:
Uhhh, super brainfart. ignore me, i completely missed the + +.
Diffs of diffs, man. Can never keep them straight in your mind

Reply:
Yeah, apologize my ineptitude. This is fine.